### PR TITLE
templates: add osrd-images templates and update values

### PR DESCRIPTION
**Pull request by younesschrifi**

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -6,7 +6,6 @@ imagePullSecrets: []
 osrdyne:
   enabled: true
   internalUrl: http://osrdyne.osrd.svc.cluster.local
-  enabled: true
```
Comment (ElysaSrc) [inline values.yaml:9]: Why removing this ?

**Review by Khoyo — APPROVED** [review]

File: values.yaml [hunk 2]
```diff
@@ -136,3 +135,21 @@ gateway:
       - "10.0.0.0/8"
       - "172.16.0.0/12"
       - "192.168.0.0/16"
+
+images:
+
```
Comment (ElysaSrc) [inline values.yaml:140]: Remove line break

File: templates/images_service.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  type: {{ .Values.images.service.type }}
+  ports:
+    - port: {{ .Values.images.service.port }}
+      targetPort: {{ .Values.images.service.targetPort }}
+      protocol: TCP
+      name: http
+  selector:
+    {{- include "osrd.selectorLabels.images" . | nindent 4 }}
+
+{{- end }}
```
Comment (Khoyo) [inline templates/images_service.yaml:19]: Missing newline at EOF ♥

File: values.yaml [hunk 4]
```diff
@@ -136,3 +135,21 @@ gateway:
...
+    type: ClusterIP
+    port: 80
+    targetPort: 80
+  annotations: {}
+  labels: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  resources: {}
+  enn: []
```
Comment (Khoyo) [inline values.yaml:155]: Ditto

**Review by Khoyo — CHANGES_REQUESTED** [review]

**Review by ElysaSrc — APPROVED** [review]
LGTM overall

**Pull request merged**